ArcadiaParties-SPA/src/app/user/user-identity/user-identity.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { Observable } from 'rxjs';
+
+import { loadUser } from '../actions/actions';
+import { State } from '../../reducers/reducers';
```

Comment:
Also it is good practice to overload base state in feature state modules.
```ts
export interface State extends rootReducers.State {
  [...FeatureKey]: ...FeatureState;
}

export const ...FeatureStateSelector = createFeatureSelector<State, ...FeatureState>(...FeatureKey);
```
and use it instead of root state to inject `Store`.